osd_cluster_admin/osd_nodes/osd-managing-worker-nodes.adoc

```diff
@@ -22,6 +22,7 @@ include::modules/rosa-osd-node-label-about.adoc[leveloffset=+1]
 .Additional resources
 
 * For more information about labels, see link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[Kubernetes Labels and Selectors overview].
+* For more information about custom additional security group requirements, see xref:../../osd_planning/aws-ccs.adoc#osd-security-groups-custom_aws-ccs[Security groups].
```

Comment:
Because you're linking to the "Additional custom security groups" section, consider changing the link text here to match that heading.

Reply:
changed